### DISCUSSION TITLE
German "Zeitungstasche" for newspaper vending maschine

### DISCUSSION THREAD

**Discussion by RoToRa** [streetcomplete/StreetComplete#2416] [Feedback]
Hi,

the German word "Zeitungstasche" that SC is using for a newspaper vending machine seems to be a specifically Austrian name for a slightly different object. (See [Sonntagsstandl](https://de.wikipedia.org/wiki/Sonntagsstandl) on Wikipedia).

I'd like to suggest changing it to the more common term "Zeitungsautomat".

Robin

**Comment by matkoniecz** [comment]
See https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app - you can change translations directly, without any programming at all (changes are pulled on releases).

Note: I have no idea at all whatever proposed change is a good idea.

**Comment by westnordost** [comment]
These things are translated at iD actually: https://github.com/openstreetmap/iD/blob/develop/CONTRIBUTING.md#translating

I changed it.

**Reply by matkoniecz** [reply]
https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app is now amended to mention also iD presets (sorry for a misleading comment, I forgot that some translations are from iD)
